src/Swashbuckle.AspNetCore.SwaggerGen/XmlComments/XmlCommentsTextHelper.cs

```diff
@@ -129,30 +135,73 @@ private static string HumanizeMultilineCodeTags(this string text)
 
         private static string HumanizeParaTags(this string text)
         {
-            return ParaTag().Replace(text, match =>
-            {
-                var paraText = "<br>" + match.Groups["display"].Value.Trim();
-                return LineBreaks().Replace(paraText, _ => string.Empty);
```

Comment:
reverting line breaks replacing to empty string, it helps to keep original `<para>` formatting